rl755/models/common/lstm.py

```diff
@@ -0,0 +1,29 @@
+import tensorflow as tf
+from rl755.models.common.sequential import SequentialModel
+
+class LSTM(SequentialModel):
+    def __init__(self, hidden_size, output_size, **kwargs):
+        super().__init__(**kwargs)
+        self.output_size = output_size
+        self.hidden_size = hidden_size
+        self.lstm = tf.keras.Sequential([
+            tf.keras.layers.Dense(units=self.hidden_size, activation=None),
+            tf.keras.layers.LSTM(units=self.hidden_size, return_sequences=True)
+        ])
+        self.final_layer = tf.keras.layers.Dense(
+            units=self.output_size, activation=None
+        )
+
+    def call(self, inputs, training=None):
```

Comment:
I think you'll need to add a `mask=None` parameter here. Due to the autoregressive nature of the LSTM, you're probably fine with doing nothing with it.